src/NuGet.Services.AzureSearch/ScoringProfiles/NewlyPublishedAndDownloadCountBoosterProfile.cs

```diff
@@ -1,33 +1,42 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
 using System.Collections.Generic;
 using Microsoft.Azure.Search.Models;
 using NuGet.Services.AzureSearch.SearchService;
 
 namespace NuGet.Services.AzureSearch.ScoringProfiles
 {
-    public class DownloadCountBoosterProfile
+    public class NewlyPublishedAndDownloadCountBoosterProfile
```

Comment:
Perhaps we should just call this `DefaultBoosterProfile` since it will iterate methodologies a lot? Or in the future will we add another class each time the approach changes?

As it stands right now, we're making a name that encodes the implementation detail which is a little unmaintainable.